tests/sentry/incidents/endpoints/test_project_alert_rule_details.py

```diff
@@ -237,6 +291,87 @@ def test_kicks_off_slack_async_job(
         }
         mock_find_channel_id_for_alert_rule.assert_called_once_with(kwargs=kwargs)
 
+    @responses.activate
+    def test_create_slack_alert_with_name_and_channel_id(self):
+        """
+        The user specifies the Slack channel and channel ID (which match).
+        """
+        channelName = "my-channel"
+        # Specifying an inputChannelID will cause the validate_channel_id logic to be triggered
+        channelID = 123
+        self._mock_slack_response(
```

Comment:
The first approach is to mock the response. I will point out the other approach once it appears.